packages/address-mocks/src/index.ts

```diff
@@ -44,9 +44,10 @@ export function mockCountryRequests() {
       const countries = fixtures.countries[locale];
       if (operationName === 'countries') return countries;
 
-      const country = countries.data.countries.find(
-        ({code}) => code === countryCode,
-      );
+      const country =
+        countryCode === 'CA'
```

Comment:
The exception in the fixtures is that only canada is available in french.

Instead of hard-coding this exception, we could dump the french response of the atlas service as `countries_fr.ts` and then, we could remove all `country_ca_xy.ts` files as they would no longer be necessary.

Reply:
This feels like a nicer fix than adding the exception!

Reply:
> If we were to remove the counyry_* mocks then I think we'd need to adjust what gets reexported

Oh right, true, let's keep them then. It can still be useful when imported as-is.